spec/models/message_spec.rb

```diff
@@ -95,7 +95,12 @@ def build_message(options = {})
 
       #note unfortunately expect :update_store is working here based on the order it's called in code
       expect(message).to receive(:send_update) do |item, user, state, channel|
-        expect(channel).to match_array(["user_#{donor.id}","user_#{subscribed_user.id}"])
+        expect(channel).to match_array(["user_#{donor.id}"])
+        expect(state).to eq("read")
+      end
+
+      expect(message).to receive(:send_update) do |item, user, state, channel|
```

Comment:
Unused block argument - `item`. If it's necessary, use `_` or `_item` as an argument name to indicate that it won't be used.<br>Unused block argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used.